lib/web_ui/test/golden_tests/engine/canvas_to_picture_test.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:html' as html;
+
+import 'package:ui/ui.dart';
+import 'package:ui/src/engine.dart';
+import 'package:test/test.dart';
+
+import 'package:web_engine_tester/golden_tester.dart';
+
+void main() async {
+  final Rect region = Rect.fromLTWH(0, 0, 500, 500);
+
+  setUp(() async {
+    debugShowClipLayers = true;
+    SurfaceSceneBuilder.debugForgetFrameScene();
+    for (html.Node scene in html.document.querySelectorAll('flt-scene')) {
+      scene.remove();
+    }
+
+    await webOnlyInitializePlatform();
+    webOnlyFontCollection.debugRegisterTestFonts();
+    await webOnlyFontCollection.ensureFontsLoaded();
+  });
+
+  test('Convert Canvas to Picture', () async {
+    final SurfaceSceneBuilder builder = SurfaceSceneBuilder();
+    final Picture testPicture = await _drawTestPictureWithCircle(region);
+    builder.addPicture(Offset.zero, testPicture);
+
+    html.document.body.append(builder
+        .build()
+        .webOnlyRootElement);
+
+    await matchGoldenFile('canvas_to_picture.png', region: region, write: true);
```

Comment:
For this to pass on CI you'll need to remove `write: true`. However, you will also need to add the new screenshot file to https://github.com/flutter/goldens (example PR: https://github.com/flutter/goldens/pull/103).

If that's too much ceremony, feel free to comment out this line in this PR, and I can do the dance with the screenshots in a follow up PR.

Reply:
I simply can't get the screenshot. The test doesn't run on either Windows or Linux for me locally (on Windows by design, on Linux it should work but it doesn't, Chrome starts and then nothing ever happens).

Reply:
Commenting out renders the test inactive. If it's OK like this, let's do so.

Reply:
Filed https://github.com/flutter/flutter/issues/63711 to follow up later.